osbrain/nameserver.py

```diff
@@ -83,9 +83,9 @@ def __init__(self, addr=None, base=NameServer):
             addr = '127.0.0.1:%s' % addr
         self.addr = addr
         self.host, self.port = address_to_host_port(addr)
-        self.shutdown_event = multiprocessing.Event()
+        self._shutdown_event = multiprocessing.Event()
         self.uri = None
-        self.queue = multiprocessing.Queue()
+        self._queue = multiprocessing.Queue()
```

Comment:
We could probably hide all attribute in the NameServerProcess as well. Couldn't we?

Reply:
I'm going to leave host, port and address public and hide the rest. They are not variables that should be changed by the user, but they might want to check their value in some cases.